### ISSUE TITLE
[wish] Allow to select model, which is already selected

### ISSUE THREAD

**Issue by azaz44** [FrSkyRC/ETHOS-Feedback-Community#2643]
I think it would be great, if we could re-select current model. So in other words be able to press-and-hold on "Extra 1"

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/42830563/ebe2c8f8-cd29-48d2-8236-5018e3c2d605)

(or choose "Set Current Model" in menu for "Extra 1")

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/42830563/bf7ee7fd-6113-482a-88f2-e115dc8f47b8)

although it is already "current". And have Ethos re-select it, making full reset, preflight checks etc.

There are two reasons:
- It seems "Reset Flight" function is a bit tricky to get perfect. We could use it before "Reset Flight" issues get ironed out (instead of power on/off cycle, or reselecting the model by selecting other model first).
- Sometimes it is a better workflow. If you fly with a couple of models, doing 1-3 flights with each, then 1-3 flights with another one, and having some breaks between flights, you sometimes don't know (or care) what was the model, that you flown previously. You just want to prepare radio for the model you want to fly, go through preflight checks, power the model etc. But at the moment, steps to "prepare radio" are different depending if the model you want to fly is selected in radio or not.

**Comment by bsongis-frsky** [comment]
First reason not accepted. Reset Flight has to work correctly

**Comment by bsongis-frsky** [comment]
Second reason not understood. Your models have a name and the name is displayed!

**Comment by azaz44** [comment]
Ok, let me clarify 2nd reason. Although I keep my fingers crossed for reliable "Reset Flight".
Steps:
- I fly helicopters. This means many flights with short times (~5 minutes), multiple models flown multiple times in any order
- I leave my radio on between flights, but display goes off (I configured my radio, so display is on when model has battery = we have telemetry, otherwise display goes off)
- I take a charged battery for one of the models, and attempt to prepare the radio for this model, which means:
  - with current version:
    - wakeup the screen
    - see what model is active in the radio (which I forgot because I was chatting or browsing github in between). Depending on this:
      - if I'm flying same model again, I need to long-press the wheel and select "Reset flight"
      - if I'm flying a different model, I need to long-press MDL, select model I want to fly and long press wheel
    - so procedure is different depending on what I was flying before
  - if we would unblock "set current model" for current model, procedure could be the same:
    - wake up the screen
    - long-press MDL, select model, long press wheel. Done without checking what was flown before.

So basically just more repeatable procedure.
This would be only good for some cases => if you change models often. If you fly one model 5x then of course "Reset Flight" is better.

As I said, if "Reset Flight" works, this request is not super required, although still useful I would say.

**Comment by azaz44** [comment]
I still think this would be a great feature. I find it many times, that I approach a model after a little break, and I have to wake up and check the screen to see, if I need to do 'reset flight' or 'select model'. A repeatable procedure to make the flight would be great. And reselecting the model isn't really much work comparing to 'reset flight'.

BTW. I would love to have special functions for both 'reset flight' and 'select model'. Both are hard to access and stay in the way.

**Comment by azaz44** [comment]
Something is still needed. After one summer with Ethos I have to say that it is super great. But the most clunky part is selecting model / resetting flight before a new flight. Any improvement in this area is welcome.

**Comment by azaz44** [comment]
Still valid
